src/node_input_factory/node_input_factory.py

```diff
@@ -2,27 +2,40 @@
                                                    SteeringNodeInput,
                                                    CoordinationNodeInput,
                                                    EngineNodeInput,
-                                                   TemperatureNodeInput)
+                                                   TemperatureNodeInput,
+                                                   ServoNodeInput)
 
 
 class NodeInputFactory:
     def __init__(self):
         pass
 
-    def create_distance_node_input(self, distance, name, node_purpose_name):
-        return DistanceNodeInput(distance, name, node_purpose_name)
+    def create_distance_node_input(self, distance: float,
+                                   name: str, node_purpose_name: str):
+        return DistanceNodeInput(distance,
+                                 name, node_purpose_name)
 
-    def create_steering_node_input(self, steering_radius,
-                                   name, node_purpose_name):
-        return SteeringNodeInput(steering_radius, name, node_purpose_name)
+    def create_steering_node_input(self, steering_radius: float,
+                                   name: str, node_purpose_name: str):
+        return SteeringNodeInput(steering_radius,
+                                 name, node_purpose_name)
 
-    def create_coordination_node_input(self, location, name,
-                                       node_purpose_name):
+    def create_coordination_node_input(self, location, name: str,
```

Comment:
ik twijfel of `coordination` wel de juiste benaming is nu. Ik vertaal dat naar "uitvoering" en niet "locatiebepaling"... Misschien beter `localisation_node`?

Reply:
ik zal het veranderen naar localisation

Reply:
localization*